Makefile

```diff
@@ -0,0 +1,35 @@
+.PHONY: help
+help:
+	@echo 'Usage:'
+	@echo '  make <target>'
+	@echo ''
+	@echo 'Targets:'
+	@awk 'match($$0, /^([a-zA-Z_\/-]+):.*?## (.*)$$/, m) {printf "  \033[36m%-30s\033[0m %s\n", m[1], m[2]}' $(MAKEFILE_LIST) | sort
+
+BASE_CONTAINER_IMAGE_NAME?=registry.fedoraproject.org/fedora
+BASE_CONTAINER_IMAGE_TAG?=41
```

Comment:
Fedora 41 unit tests are failing right now because of tests trying to depsolve using dnf5 with an sbom request, which isn't supported.

Reply:
This is overridden by the GH action anyway - but I can make fedora 40 the default, for now

Reply:
> This is overridden by the GH action anyway

Sure, but when people run `make gh-action-test` locally, it will fail.